### ISSUE TITLE
Issues with groups reported by AU Presses

### ISSUE THREAD

**Issue by babaklar** [mlaa/commons#928]
AU Presses has noticed some changes in their groups since containerization. I'm copying and pasting their notes here. I have already answered the question about the site admin issue, and added the issue on post formatting to ISSUE LINK.

From Kate: 
As we've been using Groups since the refresh a couple of weeks ago, we've been experiencing some other issues of note. 

1. Events calendar: The option to submit a new event is unworkable at the moment. The calendar to select dates doesn’t pull up, there’s no option to select Groups, and there’s no submit button.
2. Discussion tab: Now that it's called “Forum,” we can certainly change the language we use to talk to members about Groups. But we want to confirm that this change is intended and stable before we do so.
3. Discussion topic posting: The authoring interface has changed a bit. The preview window is no longer available, and the published post will have tagging visible if it includes copy-pasted text—even if you don’t see the tags while you’re composing in the “Visual” setting.
4. Accessibility: The new color scheme has made certain elements of Group functions visually inaccessible. The blue and green that now show up at the top of a discussion thread makes several key pieces of text unreadable. This is the case at a very specific time—just as soon as you’ve posted a discussion topic and it’s gone live. The background to the post title, “voices” and “replies” block, and the right sidebar are all a shade of blue that makes the text difficult to read. For example, the mint green is faint when used as text (as in “replies”) and dark grey text on top of the blue and mint green doesn’t offer a lot of contrast. I’ve attached a screenshot capturing this moment.
5. Group site: Exploring the Board Group to identify other possible changes, Brenna clicked on “Site,” expecting to see that there wasn’t one. Instead, she got a warning that the site was only available to administrators of the site even though her account is an admin account and her role in the Group is as an administrator. (I can view the site.) In Group management, the “group blog” is turned on—even though we had not enabled it—with Group admin set as a default administrator of the related site.

**Comment by mikethicke** [comment]
1. There was a bug in the events plugin. It has been corrected.

2. I do remember it being called "Discussion" on some sites and "Forum" on others at some point. I'm not sure why that's no longer the case. "Forum" is the BBPress default and IMO we should just officially stick with that.

3a) The bbp live preview plugin was buggy and hadn't been updated in 6 years. I removed it when containerizing and am kind of surprised anyone noticed.

3b) There was a bug with the visual editor that was preventing pasted text from being converted properly. The bug has been corrected. Hopefully this also addresses the issue with people getting HTML tags in emails.

4) We (or I anyways) overlooked the fact that the CSS changes for the KC rebrand would also affect other organizations. We could possibly narrow the scope of the CSS selectors to make this not the case... It might be worth checking color contrast for disucsion posting on KC as well.

5) The board group has an admin-only site from 2017. It is only accessible to administrators of the *site*, which is not the same as admins of the *group*. I can delete the site.
